src/components/Recidiviz/RecidivizForm/RecidivizForm.css

```diff
@@ -1,10 +1,10 @@
 @font-face {
 	font-family: 'nlicons';
-	src:url('../static/fonts/nlicons/nlicons.eot');
-	src:url('../static/fonts/nlicons/nlicons.eot?#iefix') format('embedded-opentype'),
-		url('../static/fonts/nlicons/nlicons.woff') format('woff'),
-		url('../static/fonts/nlicons/nlicons.ttf') format('truetype'),
-		url('../static/fonts/nlicons/nlicons.svg#nlicons') format('svg');
+	src:url('../../../../static/fonts/nlicons/nlicons.eot');
```

Comment:
Same question here as elsewhere about relative imports, though the solution here would presumably be different.

Reply:
See comment above on webpack aliasing.